python/paddle/v2/framework/tests/op_test.py

```diff
@@ -398,6 +333,7 @@ def check_grad(self,
         op_attrs = self.attrs if hasattr(self, "attrs") else dict()
         self.op = create_op(self.scope, self.op_type, op_inputs, op_outputs,
```

Comment:
We are still using `create_op` method, and numeric_grads are calculated based on runtime. Maybe we need to add a TODO, to remove it someday.